frontend/src/app/login-form/login-form.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { HttpService, LoginResponse } from './../services/http.service';
+
+@Component({
+  selector: 'app-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.scss']
+})
+export class LoginFormComponent {
+  hide = true;
+  token: string = null;
+
+  constructor(private httpService: HttpService) {}
+
+  loginHandler(username: string, password: string) {
+    this.httpService.getToken(username, password).subscribe({
+      next: (data: LoginResponse) => {
+        this.token = data.token;
+        console.log(this.token);
+      },
+      error: (err) => { console.log(err.error); },
```

Comment:
Maybe render a proper error message for user? Snackbar is really cool for that

Reply:
I made a task in `frontend` kanban board about this ;)